src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -10,6 +10,7 @@
     public static final Prefix PREFIX_PHONE = new Prefix("p/");
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
+    public static final Prefix PREFIX_SORT = new Prefix("-r");
```

Comment:
Is there a use for this? Is it a prefix used parser?

Reply:
Thanks for pointing it out. Updated.